bin/gen-crons.py

```diff
@@ -7,6 +7,7 @@
 
 HEADER = '!!AUTO-GENERATED!! Edit {template}.tmpl instead.'
 TEMPLATE_DIR = os.path.abspath(os.path.join(os.path.dirname(__file__), '..', 'etc', 'cron.d'))
+LOG_DIR = '/var/log/bedrock'
```

Comment:
I see that this dir already exists on the admin node, but perhaps we should still add a couple of lines to this script to create it if necessary?